addons/account/static/src/components/action_menus/action_menus.js

```diff
@@ -0,0 +1,24 @@
+import {_t} from "@web/core/l10n/translation";
+import { ActionMenus } from "@web/search/action_menus/action_menus";
+import { CogMenuMixin } from "@web/search/cog_menu/cog_menu";
+
+
+export class AccountActionMenus extends ActionMenus {
+
+    actionMenuTitle() {
+        return _t("Download");
+    }
+
+    async getPrintItems() {
+        const printItems = await super.getPrintItems();
+        if (this.props?.getActiveIds && this.props?.resModel === "account.move") {
```

Comment:
you always receive props, do not use optional chaining everywhere...
```suggestion
        if (this.props.getActiveIds && this.props.resModel === "account.move") {
```